tests/unit/versioning/VersionID.spec.js

```diff
@@ -9,83 +10,124 @@ function randkey(length) {
-    });
+        it('should encode and decode versionIds', () => {
+            const encoded = vids.map(vid => VID.hexEncode(vid));
+            const decoded = encoded.map(vid => VID.hexDecode(vid));
+            assert.strictEqual(vids.length, count);
+            assert.deepStrictEqual(vids, decoded);
+        });
 
-    it('simple base62 version test', () => {
-        const vid = '98376906954349999999RG001  145.20.5';
-        const encoded = VID.base62Encode(vid);
-        assert.strictEqual(encoded, 'aJLWKz4Ko9IjBBgXKj5KQT2G9UHv0g7P');
-        const decoded = VID.base62Decode(encoded);
-        assert.strictEqual(vid, decoded);
-    });
+        it('should encode and decode correctly with legacy format', () => {
+            const encoded = vids.map(vid => VID.encode(vid));
+            const decoded = encoded.map(vid => VID.decode(vid));
 
-    it('base62 version test with smaller part1 number', () => {
-        const vid = '00000000054349999999RG001  145.20.5';
-        const encoded = VID.base62Encode(vid);
-        const decoded = VID.base62Decode(encoded);
-        assert.strictEqual(vid, decoded);
+            assert.strictEqual(vids.every(x => x.length > 27), true);
+            assert.strictEqual(encoded.every(x => x.length > 32), true);
+            assert.deepStrictEqual(vids, decoded);
+        });
     });
 
-    it('base62 version test with smaller part2 number', () => {
-        const vid = '98376906950000099999RG001  145.20.5';
-        const encoded = VID.base62Encode(vid);
-        const decoded = VID.base62Decode(encoded);
-        assert.strictEqual(vid, decoded);
-    });
 
-    it('base62 version test with smaller part3', () => {
-        const vid = '98376906950000099999R1  145.20.5';
-        const encoded = VID.base62Encode(vid);
-        const decoded = VID.base62Decode(encoded);
-        assert.strictEqual(vid, decoded);
-    });
+    describe('Short IDs', () => {
+        process.env.USE_HEX_VERSION_ID = 'no';
+        const vids = generateRandomVIDs(count);
 
-    it('base62 version test with smaller part3 - 2', () => {
-        const vid = '98376906950000099999R1x';
-        const encoded = VID.base62Encode(vid);
-        const decoded = VID.base62Decode(encoded);
-        assert.strictEqual(vid, decoded);
-    });
+        it('sorted in reversed chronological and alphabetical order', () => {
+            for (let i = 0; i < count; i++) {
+                if (i !== 0) {
+                    assert(vids[i - 1] > vids[i],
+                        'previous VersionID is higher than its next');
+                }
+            }
+        });
 
-    it('error case: when invalid base62 key part 3 has invalid base62 character', () => {
-        const invalidBase62VersionId = 'aJLWKz4Ko9IjBBgXKj5KQT.G9UHv0g7P';
-        const decoded = VID.base62Decode(invalidBase62VersionId);
-        assert(decoded instanceof Error);
-    });
+        it('simple base62 version test', () => {
```

Comment:
base62Encode/Decode and hexEncode/Decode don't depend on USE_HEX_VERSION_ID, so they should be moved to their own section as was previously done.